code/tyler/django/tyler_django_lab-2/blog_proj/templates/blog/create_post.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %} {% block content %}
+<div class="d-flex col-12 justify-content-center">
+  <div class="card">
+    <form action="{%url 'blog_app:create_post'%}" method="POST">
+      <label for="blog_title">Title</label>
+      <input type="text" name="blog_title" id="blog_title" />
+      <br />
+      <label for="blog_body">Body</label>
+      <input type="text" name="blog_body" id="blog_body" />
```

Comment:
The standard convention for HTML attributes is to use kebob-case for their values. The form's `action` attribute would still use snake_case because that's how the url was written, but the `for`, `name` and `id` attributes would get switched to kebob-case